src/js/bootstrap.js

```diff
@@ -1,63 +1,89 @@
 
-const initialAccount = localStorage.getItem(ACTIVE_REMOTE_REQUEST);
-if (Cookies.get(CROSS_ACCESS_ACCOUNT_NUMBER) && initialAccount) {
-  try {
-    const { end_date } = JSON.parse(initialAccount);
-    /**
-     * Remove cross account request if it is expired
-     */
-    if (new Date(end_date).getTime() <= Date.now()) {
-      crossAccountBouncer();
+const initializeAccessRequestCookies = () => {
+  const initialAccount = localStorage.getItem(ACTIVE_REMOTE_REQUEST);
+  if (Cookies.get(CROSS_ACCESS_ACCOUNT_NUMBER) && initialAccount) {
+    try {
+      const { end_date } = JSON.parse(initialAccount);
+      /**
+       * Remove cross account request if it is expired
+       */
+      if (new Date(end_date).getTime() <= Date.now()) {
+        crossAccountBouncer();
+      }
+    } catch {
+      console.log('Unable to parse initial account. Using default account');
+      Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
     }
-  } catch {
-    console.log('Unable to parse initial account. Using default account');
-    Cookies.remove(CROSS_ACCESS_ACCOUNT_NUMBER);
   }
-}
+};
 
-// start auth asap
-const libjwt = auth();
+const libjwtSetup = () => {
+  const libjwt = auth();
 
-function noop() {}
+  libjwt.initPromise.then(() => {
+    libjwt.jwt
+      .getUserInfo()
+      .then((...data) => {
+        sentry(...data);
+      })
+      .catch(noop);
+  });
 
-//Add redhat font to body
-document.querySelector('body').classList.add('pf-m-redhat-font');
+  return libjwt;
+};
 
-libjwt.initPromise.then(() => {
-  libjwt.jwt
-    .getUserInfo()
-    .then((...data) => {
-      sentry(...data);
-    })
-    .catch(noop);
-});
+trustarcScriptSetup();
 
-window.insights = window.insights || {};
+const App = () => {
+  const modules = useSelector(({ chrome }) => chrome?.modules);
+  const scalprumConfig = useSelector(({ chrome }) => chrome?.scalprumConfig);
+  const documentTitle = useSelector(({ chrome }) => chrome?.documentTitle);
+  const dispatch = useDispatch();
+  const [jwtState, setJwtState] = useState(false);
 
-window.insights = createChromeInstance(libjwt, window.insights);
+  useEffect(() => {
+    initializeAccessRequestCookies();
+    const libjwt = libjwtSetup();
+    libjwt.initPromise.then(() => setJwtState(true));
+
+    window.insights = createChromeInstance(libjwt, window.insights);
+
+    if (typeof _satellite !== 'undefined' && typeof window._satellite.pageBottom === 'function') {
+      window._satellite.pageBottom();
+      registerUrlObserver(window._satellite.pageBottom);
+    }
 
-if (typeof _satellite !== 'undefined' && typeof window._satellite.pageBottom === 'function') {
-  window._satellite.pageBottom();
-  registerUrlObserver(window._satellite.pageBottom);
-}
+    trustarcScriptSetup();
 
-const trustarcScript = document.createElement('script');
-trustarcScript.id = 'trustarc';
+    loadFedModules().then((response) => {
+      dispatch(loadModulesSchema(response.data));
+    });
```

Comment:
Since we're using just `data` from the response, let's break it. Also no need to have function expression, just call the dispatch directly.
```suggestion
    loadFedModules().then(({ data }) => dispatch(loadModulesSchema(data)));
```